src/app/Dice.elm

```diff
@@ -17,3 +20,33 @@ emptyCup =
 cupFromIntList : List Int -> Cup
 cupFromIntList =
     identity
+
+
+viewDie : Die -> Html msg
```

Comment:
Someone can work on die rendering here. Happy to name it something else if preferred.